api/src/routes/api.ts

```diff
@@ -70,15 +70,16 @@ router.get("/:category/:job/:image/:size", async function(req, res, next) {
 });
 
 router.delete("/:category/:job/:image/*"), async function(req, res, next) {
-    //return ImageFile.delete();
+    //TO DO
+    //Sanitize incoming parameters
     let image: string = req.params.image;
     let job = getJobFromRequest(req);
     let imageObj = job.getImage(image);
     if (imageObj !== null) {
         imageObj.delete();
         res.send(JSON.stringify( { status: 'ok' } ));
     } else {
-        res.send(JSON.stringify( { status: 'image missing' } ));
+        res.status(404).send("image missing");
```

Comment:
The JSON response seems to have gotten broken when you fixed the 404 status; it's just returning the string "image missing" now, instead of the JSON object containing a status field.